assets/js/components/ErrorHandler/index.js

```diff
@@ -39,23 +68,34 @@ class ErrorHandler extends Component {
 			return children;
 		}
 
+		const reportLink = (
+			<a href="https://wordpress.org/support/plugin/google-site-kit/" target="_blank" rel="noopener noreferrer">
+				{ __( 'Report this problem', 'google-site-kit' ) }
+			</a>
+		);
+
+		const errorInfo = (
+			// eslint-disable-next-line
+			<pre onClick={ this.errorClickHandler } style={ { overflow: 'auto' } }>
+				{ error.message + info.componentStack }
```

Comment:
Since this is a `pre` tag, let's make sure these don't potentially share the same line
```suggestion
				{ error.message }
				{ info.componentStack }
```

Also, 👍  for putting these together it's much better 🙂